monitor.js

```diff
@@ -2,15 +2,24 @@
  * Monitor
  * Pokemon Showdown - http://pokemonshowdown.com/
  *
- * Various utility functions to make sure PS is running healthy.
+ * Various utility functions to make sure PS is running healthily.
  *
  * @license MIT license
  */
 'use strict';
 
 const FS = require('./fs');
 
+/**
+ * @class TimedCounter
```

Comment:
Using `@type` for each property in the constructor is how I'd do it, and I believe what the TypeScript docs recommend.